examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -196,9 +196,13 @@ while model.clock.time < end_time
     walltime = @elapsed time_step!(model, 10, wizard.Δt)
 
     ## Print a progress message
-    @printf("i: %04d, t: %s, Δt: %s, wmax = %.1e ms⁻¹, wall time: %s\n", 
+    # @printf("i: %04d, t: %s, Δt: %s, wmax = %.1e ms⁻¹, wall time: %s\n", 
+    #         model.clock.iteration, prettytime(model.clock.time), prettytime(wizard.Δt), 
+    #         wmax(model), prettytime(walltime))           
+    @info string("i: %04d, t: %s, Δt: %s, wmax = %.1e ms⁻¹, wall time: %s\n", 
             model.clock.iteration, prettytime(model.clock.time), prettytime(wizard.Δt), 
             wmax(model), prettytime(walltime))           
```

Comment:
To keep the existing format you want `@sprintf` here, not `string`.  `string` concatenates the various values:

```julia
julia> string("%04d", 10)
"%04d10"

julia> using Printf

julia> @sprintf("%04d", 10)
"0010"
```

Reply:
That explains the literal `%s`'s that were getting printed!

Reply:
BTW, `@info` allows you to attach arbitrary keywords which can preserve a lot of semantic information for later analysis rather than trying to cram all that information into a string and then parse it again later.

For example:

```julia
@info "Ocean wind mixing iteration" clock=model.clock Δt=wizard.Δt wmax=wmax(model) walltime progress=model.clock.time/end_time
```

`MicroLogging` master can print progress bars when the `progress` keyword is attached.